### DISCUSSION TITLE
Norditrack RX800 hardware setup

### DISCUSSION THREAD

**Discussion by fabfire** [JaapvanEkris/openrowingmonitor#65] [Rower Profile Development]
Hello,

I've discovered this amazing project and I'm very interested. 😃
I own an RX800 and I see that it is fully supported but I can't figure out how to connect it to the GPIO port of the raspberry.
Where is the sensor located ? is it on the "console" with the display or do I have to open the wheel part ?
Do I need to plug (electrically) the rower to get the data send to the raspberry ?

Sorry if it is noob questions but I need to understand how to wire it. ☺️

Thank you

**Comment by JaapvanEkris** [comment]
I also owned a NordicTrack Rx800, that is all how this started :).

Moving to ORM is pretty doable. In the connection between monitor arm and flywheel housing is a two-pin connector, which can be connected to a Dupont cable. Other end are plugged in to GND and pin 17 (see https://github.com/JaapvanEkris/openrowingmonitor/blob/main/docs/installation.md ), and you are in business. I completely removed the old monitor and mounted the Raspberry Pi to the connection plate in its place.

**Comment by fabfire** [comment]
Hello,

thank you for your help, I managed to get it work.
I've used the branch V1Beta-Architecture_Revision.

I've keep the old monitor to compare with ORM values on the same training.
I have urge differences between the two (resistance damper set on position 10, as stated on RX800 documentation to get right values)

The stoke distance on the integrated monitor is about 6 meters, which is very low, even if I pull hard. This was the case before but I never give attention to it.
Comparison on the same session :
- RX800 Monitor : 500 meters on 82 strokes on 3'30, 23-24 SPM
- ORM : 500 meters on 50 strokes on 2'05, 23-24 SPM
The values provided by ORM seems much more correct. The stroke distance is around 10 meters.

Did you noticed the same with your RX800 ?

**Reply by JaapvanEkris** [reply]
> Hello,
> 
> thank you for your help, I managed to get it work. I've used the branch V1Beta-Architecture_Revision.

Yeah, that is the best version at the moment. It will become main in January (still some small bugs to remove).

> I've keep the old monitor to compare with ORM values on the same training. I have urge differences between the two (resistance damper set on position 10, as stated on RX800 documentation to get right values)

The RX800's standard monitor data is useless. The standard parameters in the ORM rowerConfig is what I ended up with after comparing the feel/pace with a Concept2in the gym. When I switched to C2 myself, I could compare side-by-sise, and noticed it was pretty close.

> Did you noticed the same with your RX800 ?

Yeah, I don't know what their reference was, but on its maximum drag it is way too heavy to be usefull, but they claim calibration there indeed. But it is way off. In all honesty, I doubt they even measure the flywheel drag etc..

**Comment by fabfire** [comment]
hello,

here is my finished setup :)
Thank for your help !
![IMG_2433](https://github.com/user-attachments/assets/4893fb4e-6e30-4992-80f3-b602922d5b30)

I've a problem : some strokes are lost, but I will open another question to discuss it
